src/adapter.js

```diff
@@ -29,6 +30,23 @@ var createQUnitStartFn = function (tc, runnerPassedIn) {
       totalNumberOfTest += 1;
       timer = new Date().getTime();
       testResult = { success: true, errors: [] };
+
+      // create a qunit-fixture element to match behaviour of regular qunit
+      // runner. The fixture is only removed at the start of a subsequent test
+      // so it can be inspected after a test run.
+      var fixture = document.getElementById(FIXTURE_ID);
+      if (fixture) {
+        fixture.parentElement.removeChild(fixture);
```

Comment:
Use `parentNode` instead to account for a wider spectrum of possible scenarios users of Karma may find themselves in.